platform-hub-api/app/serializers/user_serializer.rb

```diff
@@ -1,5 +1,5 @@
 class UserSerializer < BaseSerializer
-  attributes :id, :name, :email, :role, :last_seen_at
+  attributes :id, :name, :email, :role, :last_seen_at, :is_active
```

Comment:
Like `identities` below, we should probably make this an "admin or own" property?